get_bams/1.0/get_bams.R

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/env Rscript
+#
+# Usage:
+#   get_bams.R <input_table> <output_table> [<library_id_column>]
+#
+# Notes:
+#   - <library_id_column> can be a name or the number 1. If it is the
+#     number 1, the script assumes that the input file has no header
+#     and only has a single column. The default value is 'library_id'
+#     (i.e., the script assumes a file header).
+#
+#   - The column names are prefixed with 'al.', 'lc.', and 'lb.' for
+#     designating alignment-, libcore-, and library-specific columns.
+#     It also helps with ensuring no collisions when joining the
+#     obtained information with the input table.
+#
+
+
+# Load packages -----------------------------------------------------------
+
+suppressWarnings(suppressPackageStartupMessages({
```

Comment:
Is there a reason why this `suppressWarnings()` and `suppressPackageStartupMessages()` encompass the entire script? I would think it would be useful to have warnings being printed for the user, and they can opt out using `2>/dev/null`.

Reply:
I think it inundates the user with uninformative warnings related to e.g. `pivot` commands that are hard to interpret if you're not running things interactively. IMO the messages we've specified are sufficient, and if there are errors the user will still see messages related to those.

Reply:
I see. In that case, I'm fine with this as is.